components/checks.py

```diff
@@ -6,11 +6,11 @@
 
 
 def register_command_validated(ctx: commands.Context) -> bool:
-    return in_recruit_channel(ctx) and is_recruiter(ctx)
+    return in_recruit_channel(ctx)
```

Comment:
in_recruit_channel is something that server admins can restrict, rather than having the bot do it.  Not something that needs to be handled in this PR, though

Reply:
It is not particularly harmful to leave it in either, though. Otherwise this looks fine to me.